rxgroups/src/main/java/com/airbnb/rxgroups/ManagedObservable.java

```diff
@@ -86,25 +86,6 @@ String tag() {
     return tag;
   }
 
-  @Override public boolean equals(Object o) {
```

Comment:
Is it ok to remove the hashcode methods? Don't we still use it when saving state? or I guess it's ok to just use the default hashcode method which will tell us if the object instantiation is different?